### PR TITLE
Improve web sharing

### LINKED ISSUE #246: Improve Facebook sharing view of page
Currently looks like: 

image

Instead we want: 

* A screenshot showing our main model and the Oxford logo. I made some to the right dimensions, available in Notion [here](https://www.notion.so/Site-thumbnail-60cbc9e23fe945bfae936083b0550f73)
* The title should not be "model", but instead "Medium-term COVID-19 forecast"

### PR DESCRIPTION
**Pull request by wolverdude**
* Updated page titles
* Added site description
* Improved Facebook thumbnail (Fixes #246)
* Added share buttons

### CODE REVIEW

**Comment by wolverdude** [comment]
Not quite ready for merging just yet.

File: src/server/templates/footer.html [hunk 1]
```diff
@@ -1,37 +1,41 @@
...
+    <a class="a2a_button_whatsapp"></a>
+    <a class="a2a_dd" href="https://www.addtoany.com/share"></a>
+  </div>
+  <script async src="https://static.addtoany.com/menu/page.js"></script>
+  <!-- AddToAny END -->
+  <!-- Sponsor Links -->
+  <div>
+    <img
+      class="logo"
+      src="/static/img/Oxford-logo.png"
```
Comment (hnykda) [inline src/server/templates/footer.html]: Would be nice to put in `url_for`...

Comment (wolverdude) [reply src/server/templates/footer.html]: Good point. I'll update those and then merge.

**Comment by wolverdude** [comment]
New share thumbnail is working on staging:

Screen Shot 2020-03-25 at 1 16 40 PM

**Pull request merged**